File: Java/2701-2750/2706. Buy Two Chocolates.java
// https://leetcode.com/problems/buy-two-chocolates/

class Solution {
    public int buyChoco(int[] prices, int money) {
        int min1 = Integer.MAX_VALUE, min2 = min1;

        for(int price : prices){
            if(price < min1){
                min2 = min1;
                min1=price;
            }else if(price < min2) min2 = price;
        }

        int cost = min1 + min2;

        if(cost <= money) return money - cost;
        
        return money;
    }
}

File: Python/2701-2750/2706. Buy Two Chocolates.py
# https://leetcode.com/problems/buy-two-chocolates/

class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        min1, min2 = 99999, 99999

        for price in prices:
            if price < min1:
                min2 = min1
                min1=price
            elif price < min2:
                min2 = price
        
        cost = min1 + min2

        if(cost <= money):
            return money - cost
        
        return money
